epam-promo/components/overlays/Tooltip.scss

```diff
@@ -7,7 +7,6 @@
         font-size: 14px;
         font-family: $font-sans;
         box-shadow: 0 1px 4px 0 rgba(44, 47, 60, 0.05), 0 2px 18px 0 rgba(44, 47, 60, 0.03);
-        max-width: 300px;
```

Comment:
It seems, the same change needs to be applied to "loveship" skin as well: loveship/components/overlays/Tooltip.scss

Reply:
Thank you for your attention) Fixed